pydy/codegen/tests/test_c_code.py

```diff
@@ -105,7 +105,17 @@ def test_generate_comma_lists(self):
                     'v0(t), v1(t), v2(t), v3(t), v4(t), v5(t)',
                     'f5(t), f2(t), f3(t), f0(t), f4(t), f1(t)')
```

Comment:
I see, as per the other discussions I think that switching to sets may not be the best move as the printing code relies on the particular order of the underlying c arrays to be fixed.